sdk/nodejs/output.ts

```diff
@@ -307,6 +307,7 @@ To manipulate the value of this Output, use '.apply' instead.`);
     public apply<U>(func: (t: T) => Input<U>, runWithUnknowns?: boolean): Output<U> {
         // we're inside the modern `output` code, so it's safe to call `.allResources!` here.
 
+        runWithUnknowns = true;
```

Comment:
I think to minimise blast radius we could do a pass removing the `ifDryRun` apply logic from nodejs as it's own PR. Nothing _should_ really depend on that, and we can sanity check just that change is safe.